packages/map-template/src/components/Map/MapboxMap/MapboxMap.jsx

```diff
@@ -38,6 +39,10 @@ function MapboxMap({ onMapView, onPositionControl }) {
         const externalDirectionsProvider = new window.mapsindoors.directions.MapboxProvider(mapboxAccessToken);
 
         onMapView(mapViewInstance, externalDirectionsProvider);
+
+        // Generate a UUIDv4 and set the Session Token for searching for Mapbox places.
+        const uuid = uuidv4();
+        sessionStorage.setItem('sessionToken', uuid);
```

Comment:
The name may be a little too generic. `mapboxPlacesSessionToken`?

Reply:
Renamed the key to `mapboxPlacesSessionToken`.